app/services/medium_providers/provider.rb

```diff
@@ -4,7 +4,7 @@ module MediumProviders
   # Example:
   # provider = MyMediumProvider.new(centos_host)
   # provider.medium_uri
-  # => #<URI::HTTP http://mirror.centos.org/centos/7/os/x86_64>
+  # => #<URI::HTTP http://mirror.stream.centos.org/9-Stream/BaseOS/x86_64/os>
```

Comment:
9-Stream is not available at the mirror 
```suggestion
  # => #<URI::HTTP http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os>
```